### PR TITLE
fix: scripts

### PR DESCRIPTION
**Pull request by aarnphm**
## What does this PR address?

Fix some scripts

## Before submitting:

- [ ] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
      guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [ ] Does the code follow BentoML's code style, both `make format` and `make lint` script have passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [ ] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [ ] Did your changes require updates to the documentation? Have you updated
      those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

### CODE REVIEW

File: scripts/release_pypi.sh [hunk 1]
```diff
@@ -53,7 +53,7 @@ fi
 
 echo "Generating PyPI source distribution..."
 cd "$GIT_ROOT"
-python3 -m build -s -w
```
Comment (sauyon) [inline scripts/release_pypi.sh:56]: What's this change for?

Comment (aarnphm) [reply scripts/release_pypi.sh:56]: build by default already create wheels and source distribution, no need to pass in the arguments.

Comment (sauyon) [reply scripts/release_pypi.sh:56]: Let's leave the arguments here? I think it's nicer, if only just as documentation of what we're doing.

Comment (aarnphm) [reply scripts/release_pypi.sh:56]: sg.

**Pull request closed without merging**